main.js

```diff
@@ -35,6 +60,33 @@ function makeRequest(fn, uri, options, callback) {
     }
 }
 
+Trello.prototype.makeRequest = function (requestMethod, uri, options, callback) {
```

Comment:
Actually this `uri` is not really an `URI`. Its merely the `path` part of an URI. So maybe rename this argument to `path`?